### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by VolodymyrPst**

### CODE REVIEW

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Demo link doesn't work, try to fix it or ask for help in fe_chat

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job
Let's to fix some mistakes
Add hover effects for all clickable elements and buttons
Need to fix 
Demo
image
Design
image

**Comment by VolodymyrPst** [comment]
https://volodymyrpst.github.io/Museum/

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
No hover effects for this button, add it
Also add a `cursor: pointer` for all clickable elements
image

Clicking on the phone number does not need to open in a new tab
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Keep up the good work! Some comments to fix:

- Fix burger-menu icons. The menu and cross icons should be on the same level 
- Add cursor pointer to each clickable elements
- Remove redundant elements
image
- form don't work. I can't send my email
image
- Add hover effect to images in gallery and to links in footer 
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
You need to remove default styles for input after autocomplete
image

Add additional styles for all clickable elements when on hover
image

Remove scroll when this menu is open
image

**Review by nazarmatsevych — APPROVED** [review]
Well done!

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Looks nice! Additional comments: 
Screenshot 2023-08-17 at 23 31 36
1) Validation error
Screenshot 2023-08-17 at 23 33 43
Screenshot 2023-08-17 at 23 37 22
Screenshot 2023-08-17 at 23 37 37
Screenshot 2023-08-17 at 23 37 52
2) The styling doesn't match the Figma design
Screenshot 2023-08-17 at 23 34 08
3) Consider removing browser outlines from the button
Screenshot 2023-08-17 at 23 35 10
4) Check the styling for this text
Screenshot 2023-08-17 at 23 38 11
Screenshot 2023-08-17 at 23 38 25
5) The layout for this block seems a bit broken

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Almost there) Just several more things needs to be fixed
your landing
Screenshot at Aug 19 13-40-01

design
Screenshot at Aug 19 13-40-12

No need to do exactly same thing, just pls add some contrast to text
Also just a suggestion - consider importing icons and logo in .svg format

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,444 @@
...
+              </p>
+            </div>
+          </div>
+
+          <div class="relevant__exhibition">
+            <div class="relevant__image relevant__image--second"></div>
+            <div class="relevant__info">
+              <!-- <p class="relevant__info-date">ВИСТАВКА
+                <span class="relevant__info-date-span">26.08-29.11.2019</span>
+              </p> -->
```
Comment (volodymyr-soltys97) [inline src/index.html:130]:
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,444 @@
...
+                            href="https://goo.gl/maps/jTgxe8emqQnQ6Wc56"
+                            target="_blank"
+                            class="contacts__map"
+                          >
+                            вул. М. Грушевського, 6; Київ, 01001
+                          </a>
+                        </li>
+                        <li class="contacts__item-contact">
+                          <a
+                            href="tel:+38(044)2781357"
```
Comment (volodymyr-soltys97) [inline src/index.html:349]:
```suggestion
                            href="tel:+380442781357"
```

File: src/styles/blocks/contacts.scss [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+    @include onTablet {
+      display: flex;
+    }
+
+    @include onDasktop {
+      display: flex;
+    }
+  }
+  &__fb {
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/contacts.scss:51]: Add blank line between the selectors everywhere for better readability
```suggestion
  }
  
  &__fb {
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!

**Review by anastasiiavorobiova — APPROVED** [review]
Looks perfect!

**Pull request closed without merging**